debian/changelog

```diff
@@ -1,3 +1,10 @@
+ralph (20190109.2) bionic; urgency=medium
```

Comment:
why this change? Do we want to issue new release at the same time?

Reply:
Nope, as usual I pushed the test commit unintentionally. Currently I’m rushing the bus driver to get to my workplace and kick this commit ASAP :)